enclave-tls/src/tls_wrappers/openssl/negotiate.c

```diff
@@ -9,6 +9,8 @@
 #include <enclave-tls/tls_wrapper.h>
 #include "openssl.h"
 
+int ex_data_idx = 0;
```

Comment:
Mark as static.

Reply:
ex_data_idx serves for ctx, and ctx serves for per thread. So what happens if multiple threads call openssl_internal_negotiate()? How to keep ex_data_idx for each thread?